tests/lib/mkpinentry.sh

```diff
@@ -11,6 +11,9 @@ else
             ;;
         *)
             mkdir -p ~/.snap/gnupg/
+            # It is just created once by gpg-agent, then if this dir does not exist gpg will
+            # fail to create the key
```

Comment:
Woah, is there a bug for this?

Reply:
@zyga gpg v2.1 says in its doc that that dir is created just once, then at some point it was changed and in the version 2.2.20 (the one used in debian-sid) if the dir does not exist, it is created automatically by the gpg-agent.